jvb/src/main/java/org/jitsi/videobridge/shim/ChannelShim.java

```diff
@@ -267,7 +273,19 @@ public void describe(ColibriConferenceIQ.ChannelCommon commonIq)
      * Sets the list of sources signaled for this channel.
      */
     public void setSources(@NotNull List<SourcePacketExtension> sources)
+        throws IqProcessingException
     {
+        if (multiStreamConfig.getEnabled()) {
+            for (SourcePacketExtension s: sources) {
+                // TODO use Kotlin "isNullOrEmpty" once ported to Kotlin
+                if (s.getName() == null || s.getName().trim().isEmpty()) {
```

Comment:
@bgrozev what's the recommended way of doing "is null or empty check" now? I saw StringUtils were deprecated